src/test/testdata/expected/lsl/charLiterals.lsl

```diff
@@ -0,0 +1,13 @@
+libsl "1.0.0";
+library simple;
+typealias Int = int32;
+typealias Char = char;
+
+automaton A : Int {
+    var ch1: Char = 'A';
+    var ch2: Char = 'a';
+    var ch3: Char = 52;
+    var ch4: Char = '*';
+    var ch5: Char = '';
```

Comment:
This line doesn't seem correct to my eye.

Reply:
Was deleted